src/theme/components/card.ts

```diff
@@ -247,4 +255,4 @@ const CardImage = {
   },
 };
 
-export default { Card, CardActions, CardContent, CardImage };
+export default { CustomCard, CardActions, CardContent, CardImage };
```

Comment:
For consistency, update `CardActions`, `CardContent`, and `CardImage` with a similar update as `CustomCard`. All of these use `useStyleConfig` so you won't need to use `defineMultiStyleConfig`. Here's an example: https://github.com/NYPL/nypl-design-system/blob/react18-and-chakra/src/theme/components/buttonGroup.ts